libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -89,6 +102,18 @@ class EmbeddingStore(Base):
     def embeddings(self) -> Embeddings:
         return self.embedding_function
 
+    @property
+    def distance_strategy(self) -> str:
+        # Value of distance strategy passed in should be one of the supported values.
+        if self._distance_strategy == DistanceStrategy.EUCLIDEAN_DISTANCE:
+            return "euclidean"
```

Comment:
We can return the  DistanceStrategy.EUCLIDEAN_DISTANCE instead. Or store create constants for the "euclidean", "cosine" and "dot" strings, and return the constants, rather than returning strings directly.

Reply:
The literal values from DistanceStrategy class are different from the expected literal string in TSQL. We can have our own DistanceStrategy class instead and that will help with reusing variables.

Reply:
Yeah, let us add our own `DistanceStrategy` class, I think we can use the `utils.py` file for that

Reply:
Nah, that file isn't owned or managed by us, and our definition of DistanceStrategy isn't necessarily going to be usable for others. There also is an already existing Default DistanceStrategy class provided too.

However, we can manage ours in-house (i.e. in our file).

Reply:
Oh okay, got it

Reply:
Done.